doc/manual/configuration.xml

```diff
@@ -710,27 +710,6 @@ basedir regex = /usr/home</programlisting></para>
             to your configure switches to have this UAM available.</para>
           </listitem>
 
-          <listitem>
-            <para>"Kerberos IV"<indexterm>
-                <primary>Kerberos IV</primary>
-
-                <secondary>"Kerberos IV" UAM</secondary>
-              </indexterm><indexterm>
-                <primary>uams_krb4.so</primary>
-
-                <secondary>"Kerberos IV" UAM</secondary>
-              </indexterm>/"AFS Kerberos"<indexterm>
-                <primary>AFS Kerberos</primary>
-
-                <secondary>"AFS Kerberos" UAM (Kerberos IV)</secondary>
-              </indexterm> UAMs (suitable to use <ulink
-            url="http://web.mit.edu/macdev/KfM/Common/Documentation/faq.html">Kerberos
-            v4 based authentication</ulink> and AFS file servers)</para>
-
-            <para>Use <filename>"--enable-krb4-uam"</filename> at compile time
-            to activate the build of this UAM.</para>
-          </listitem>
-
```

Comment:
Isn't removing all the Kerberos docs overkill? We still support the stock UAMs.

Reply:
Have just removed Kerberos IV UAM reference as --enable-krb4-uam is not an option for Netatalk 3 (only Kerberos V support here, --enable-krbV-uam)

Reply:
Aha, I had forgotten about that, after using netatalk2 too much. :)